dist/profile/templates/confluence/vhost.conf

```
@@ -1572,6 +1572,7 @@ RewriteRule "^/display/JENKINS/Installing\+Jenkins\+with\+Docker$" "https://www.
 RewriteRule "^/display/JENKINS/Plugins$" "https://plugins.jenkins.io/" [NE,NC,L,QSA,R=301]
 RewriteRule "^/display/JENKINS/How\+to\+fix\+RequireUpperBoundDeps$" "https://www.jenkins.io/doc/developer/plugin-development/updating-parent/#understanding-requireupperbounddeps-failures-and-fixes" [NE,NC,L,QSA,R=301]
 RewriteRule "^/display/JENKINS/Release\+Process$" "https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#release-process" [NE,NC,L,QSA,R=301]
+RewriteRule "^/display/JENKINS/Email-ext+Recipes$" "https://github.com/jenkinsci/email-ext-plugin/blob/master/docs/recipes.adoc" [NE,NC,L,QSA,R=301]
```

Need to escape the 'plus' character in the pattern match.
```suggestion
RewriteRule "^/display/JENKINS/Email-ext\+Recipes$" "https://github.com/jenkinsci/email-ext-plugin/blob/master/docs/recipes.adoc" [NE,NC,L,QSA,R=301]
```

---

oh, thanks!

---

Fixed